benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -397,6 +368,35 @@ msgstr ""
 "To function properly, this website requires a browser that supports "
 "JavaScript. Please enable JavaScript for this website and"
 
+msgid "eligibility.pages.index.login_gov.label"
+msgstr "I am 65 years of age or older"
+
+msgid "eligibility.pages.index.login_gov.description"
+msgstr ""
+"You must be 65 years or older. This benefit does not expire, but you may "
+"need to renew. Using this benefit means your new transit fare is half of the "
+"standard fare."
+
+msgid "eligibility.pages.index.mst_cc.label"
+msgstr "I have an MST Courtesy Card"
+
+msgid "eligibility.pages.index.mst_cc.description"
+msgstr ""
+"This option is for people who have a current Courtesy Card or an MST RIDES "
+"Eligibility card. This benefit may need to be renewed in the future. Using "
+"this benefit means your new transit fare is half of the standard fare."
+
+msgid "eligibility.pages.index.veteran.label"
+msgstr "US Veteran"
+
+msgid "eligibility.pages.index.veteran.description"
+msgstr ""
+"This option is for people who have served in the active military, naval, or "
+"air service, and who were discharged or released therefrom under conditions "
+"other than dishonorable. You will need to <a href='https://www.va.gov/"
+"resources/verifying-your-identity-on-vagov/' target=\"_blank\" rel="
+"\"noopener noreferrer\">verify your identity through VA.gov</a>"
+
```

Comment:
I don't know why `makemessages.sh` insists on moving these.. but it moved them the same for both En and Es files.